src/index.ts

```diff
@@ -249,14 +246,16 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
       },
     );
     instance.setConfigurationStore(configurationStore);
-    // instantiate and init assignment cache if needed
+
+    // instantiate and init assignment cache
     const assignmentCache = assignmentCacheFactory({
       chromeStorage: chromeStorageIfAvailable(),
       storageKeySuffix,
     });
     if (assignmentCache instanceof HybridAssignmentCache) {
       await assignmentCache.init();
     }
```

Comment:
This assignment cache was being instantiated, initialized but not set to the singleton client.

Reply:
welp my bad